sql/statesql/statesql.go

```diff
@@ -0,0 +1,54 @@
+package statesql
+
+import (
+	"embed"
+
+	"github.com/spacemeshos/go-spacemesh/sql"
+)
+
+//go:embed schema/schema.sql schema/migrations/*.sql
+var embedded embed.FS
```

Comment:
If you embed the `schema.sql` file directly some code can be simplified:

```suggestion
//go:embed schema/schema.sql
var schema string

//go:embed schema/migrations/*.sql
var migrations embed.FS
```

Reply:
Done